apps/graphql/src/dataloaders/BookingsLoader.js

```diff
@@ -57,9 +60,26 @@ const sanitizeBookings = (
     arrival: sanitizeRouteStop(last(booking.flights).arrival),
     departure: sanitizeRouteStop(head(booking.flights).departure),
     passengerCount: booking.passengers.length,
+    type: detectType(booking),
   }));
 };
 
+const detectType = (booking: BookingApiResult) => {
+  if (Array.isArray(booking.flights)) {
+    const returnFlight = booking.flights.find(flight => flight.return === 1);
```

Comment:
Okay, I just find `return` of type number to be confusing.

Reply:
Yeah, could be derived from `C` or something 🤷‍♂️